build.fsx

```diff
@@ -126,14 +98,12 @@ Target.create
     "Pack"
     (fun _ ->
         let properties =
-            [ ("Version", nugetVersion)
-              ("Authors", authors)
+            [ ("Authors", authors)
               ("PackageProjectUrl", gitUrl)
```

Comment:
you can remove this project url as well - it defaults to the repo git url that sourcelink derives from the `origin` uri